app/1.0/docs/devguide/data-system.md

```diff
@@ -268,7 +268,7 @@ because it requires extra work to retrieve the index. Polymer provides methods t
 original array item given the item's key.
 
 **In most cases, you don't need to deal with array keys directly**. Polymer provides helper
-elements, such as the [template repeater element](/1.0/docs/devguide/templates#dom-repeat) that
+elements, such as the [template repeater element](/1.0/docs/devguide/templates#dom-repeat), that
```

Comment:
Here, **that** introduces a [restrictive clause](http://www.quickanddirtytips.com/education/grammar/which-versus-that-0) (not all helper elements are involved with arrays). So no comma.